src/domain/entities/CustomText.ts

```diff
@@ -0,0 +1,30 @@
+import i18n from "../../locales";
+import { getKeys } from "../../types/utils";
+import { TranslatableText } from "./TranslatableText";
+
+export interface CustomText {
+    root_title: TranslatableText;
+    root_subtitle: TranslatableText;
+}
+
+export type CustomTextInfo = { [K in keyof CustomText]: string };
+
+export const defaultCustomText: CustomText = {
+    root_title: {
+        key: "root_title",
+        referenceValue: i18n.t("Welcome to training on DHIS2"),
```

Comment:
Outside components and use hooks, i18n has not been initialised and therefore translations will not take place. To address this you could pass i18n with a wrapper.